README.md

```diff
@@ -1,24 +1,75 @@
-# README
+# Project Name
+Transaction app 
+The Transaction app will be a classic example of a Transaction website. We created a fully functional website that will show the list of groups and Transaction and empower user to interact with them 
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+## Built With
 
-Things you may want to cover:
+* Ruby on Rails
+* Rspec
+* Psql
 
-* Ruby version
+## Video
+This is the link to introductory Video: [Video](https://www.loom.com/share/a315c9a60e4249038aeb5b972461e529)
 
-* System dependencies
 
-* Configuration
+## Live Link
+This is the link to heroku link: [Link](https://vic-transaction.herokuapp.com/)
 
-* Database creation
 
-* Database initialization
+## Getting Started
 
-* How to run the test suite
+### Prerequisites
 
-* Services (job queues, cache servers, search engines, etc.)
+* Ruby (ruby --v)
+* SQLite3 (sqlite3 --v)
+* Postgrestsql (psql --version)
+* Node.js (node --v)
+* Yarn (yarn --v)
 
-* Deployment instructions
+To get a local copy up and running follow these simple example steps.
 
-* ...
+Run in your terminal `git clone https://github.com/VICTOR-CHUKWU/rails-recipe-app`
+
+### Install RSpec
+
+To run the tests locally :
+
+* To instal RSpec for testing please run the following command on your terminal:
+
+ ` gem install rspec`
+
+### Helpful links to solve some issues
+
+* [Could not find gem](https://stackoverflow.com/questions/32491201/could-not-find-gem-pg-0-12-4-ruby-in-any-of-the-gem-sources-listed-in-your)
+* [Install psql](https://harshityadav95.medium.com/postgresql-in-windows-subsystem-for-linux-wsl-6dc751ac1ff3)
+* [PG:: InsufficientPrivilege: ERROR:  must be owner of database blog_app_development](https://stackoverflow.com/questions/25610753/activerecordstatementinvalid-pgerror-error-must-be-owner-of-database)
+
+
+## Run the application
+-connect to your psql
+-  run `rails s` in your terminal
+
+## Authors
+👤 **Victor**
+
+- GitHub: [@VICTOR-CHUKWU](https://github.com/VICTOR-CHUKWU)
+- LinkedIn: [Victor-chukwu](https://www.linkedin.com/in/victor-chukwu-95a020143)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
```

Comment:
Kindly give appropriate credit to [Gregoire Vella on Behance](https://www.behance.net/gregoirevella) the [author of the original design](https://www.behance.net/gallery/19759151/Snapscan-iOs-design-and-branding?tracking_source=), as required by the [Creative Commons license of the design](https://creativecommons.org/licenses/by-nc/4.0/) in the "Acknowledgements" section.